src/structs/Api.ts

```diff
@@ -91,55 +110,52 @@ export class Api extends EventEmitter {
    * @example
    * ```js
    * await api.postStats({
-   *   serverCount: 28199,
-   *   shardCount: 1,
+   *   serverCount: 28199
    * });
    * ```
    *
    * @param {object} stats Stats object
    * @param {number} stats.serverCount Server count
-   * @param {number} [stats.shardCount] Shard count
-   * @param {number} [stats.shardId] Posting shard (useful for process sharding)
    * @returns {BotStats} Passed object
    */
   public async postStats(stats: BotStats): Promise<BotStats> {
-    if (!stats?.serverCount) throw new Error("Missing Server Count");
+    if ((stats?.serverCount ?? 0) <= 0) throw new Error("Missing server count");
 
     /* eslint-disable camelcase */
     await this._request("POST", "/bots/stats", {
       server_count: stats.serverCount,
-      shard_id: stats.shardId,
-      shard_count: stats.shardCount,
     });
     /* eslint-enable camelcase */
 
     return stats;
   }
 
   /**
-   * Get a bots stats
+   * Get your bot's stats
    *
    * @example
    * ```js
-   * await api.getStats("461521980492087297");
+   * await api.getStats();
    * // =>
    * {
    *   serverCount: 28199,
-   *   shardCount 1,
+   *   shardCount: null,
    *   shards: []
    * }
    * ```
    *
-   * @param {Snowflake} id Bot ID
-   * @returns {BotStats} Stats of bot requested
+   * @returns {BotStats} Your bot's stats
    */
-  public async getStats(id: Snowflake): Promise<BotStats> {
-    if (!id) throw new Error("ID missing");
-    const result = await this._request("GET", `/bots/${id}/stats`);
+  public async getStats(_id?: Snowflake): Promise<BotStats> {
```

Comment:
We should still support the possibility of passing in an ID other than the bot that is authenticated.
We shouldn't need to do anything to this endpoint, instead just default id to the id generated from the client so it's possible to use as just getStats()

Reply:
[Not possible by the API.](https://discord.com/channels/264445053596991498/412006692125933568/1348359048403353610)